IDE_Board_Manager/prusa3dboards/platform.txt

```diff
@@ -25,14 +25,14 @@ compiler.c.elf.flags=-Os -g -flto -Wl,--gc-sections
 compiler.c.elf.cmd=avr-gcc
 compiler.S.flags=-c -g -x assembler-with-cpp -flto -MMD
 compiler.cpp.cmd=avr-g++
-compiler.cpp.flags=-c -g -Os {compiler.warning_flags} -std=gnu++11 -ffunction-sections -fdata-sections -MMD -flto -fno-fat-lto-objects -fno-exceptions -fno-threadsafe-statics -fno-rtti
+compiler.cpp.flags=-c -g -Os {compiler.warning_flags} -std=gnu++17 -ffunction-sections -fdata-sections -MMD -flto -fno-fat-lto-objects -fno-exceptions -fno-threadsafe-statics -fno-rtti
 compiler.ar.cmd=avr-gcc-ar
```

Comment:
I cannot add a comment on the exact line above, but on ``compiler.c.flags`` I would also set -std=gnu17 to match the same standard for both c and c++.

Reply:
I tried that and I think that is not possible

Reply:
Yeah, just checked now. Looks like gcc7 doesn't support it. Well then, nothing we can do about this last point.